src/Mangata.ts

```diff
@@ -113,37 +123,33 @@ export class Mangata {
    */
 
   public async createPool(
-    account: KeyringPair | string,
-    firstAssetId: string,
-    firstAssetAmount: BN,
-    secondAssetId: string,
-    secondAssetAmount: BN,
+    account: MangataAccount,
+    firstTokenId: TokenId,
+    firstTokenAmount: Amount,
+    secondTokenId: TokenId,
+    secondTokenAmount: Amount,
     txOptions?: TxOptions
   ): Promise<MangataGenericEvent[]> {
+    log.info(`Creating pool ...`)
     const api = await this.connect()
```

Comment:
Do we really need to `connect()` on every action instead of connecting once before using the api?
If yes, what is the reason?
If no, please refactor.

Reply:
maybe the word connect is not correctly used here.. it should be getApi I think.. in this method we are not connecting .. we just getting API thats all.. I will refactor.